[16.0][WIP] rma_account_unreconciled: Migration to 16.0

AlexPForgeFlow opened the pull request:
Migration to 16.0

**(Features pending of new version)** This module allows to  reconcile those Orders in a single click. In accounting
settings users will be able to set up a specific account for write-off.

[ForgeFlow](https://github.com/ForgeFlow)

ChrisOForgeFlow committed:
[14.0][ADD] rma_account_unreconciled

AaronHForgeFlow committed:
[FIX] rma: rma_custmer_user has no write permissions in partner, so compute method fails.
[IMP] rma: use rma user in tests
[FIX] rma_account: move_line_id field string
[IMP] rma, rma_account, rma_sale, rma_purchase: tests for stock valuation
[FIX] account_move_line_rma_order_line: minor lint, make auto-install
[IMP] rma_account_unreconciled: use manager user in test

JordiBForgeFlow committed:
make module AGPL

DavidJForgeFlow committed:
[FIX]rma: remove test_rma dependency to Account

JordiBForgeFlow committed:
[FIX] include anglo-saxon price unit calculation in refunds.
Otherwise the anglo saxon entries won't be correct.
For example, the Interim (Delivered) account should balance
after receiving and triggering a refund on a customer rma.

AlexPForgeFlow committed:
[IMP] rma_account_unreconciled: pre-commit stuff

AlexPForgeFlow committed:
[MIG] rma_account_unreconciled: Migration to 16.0

AaronHForgeFlow commented on rma_account_unreconciled/views/rma_line_view.xml:
```diff
@@ -0,0 +1,40 @@
+                    type="object"
+                    string="Unreconciled Journal Items"
+                    name="action_view_unreconciled"
+                    groups="account.group_account_invoice"
```
```suggestion
                    groups="account.group_account_user"
```
Because standard Odoo does not let access to journal items if not in that group

AaronHForgeFlow reviewed (commented):
Works good :+1: 

CI red just because dependencies not merged.

BernatPForgeFlow committed:
[FIX] rma_account_unreconciled: Remove 'account_reconcile_oca' dependency

RMA lines should be able to reconcile either with OCA or Enterprise reconcile tools.
